app/assets/javascripts/tylium/behaviors.js.coffee

```diff
@@ -160,33 +160,37 @@ document.addEventListener "turbolinks:load", ->
 
   # Toggle sidebar menu
 
+  $navbar = $('[data-behavior~=navbar]')
   $sidebar = $('[data-behavior~=main-sidebar]')
-
-  sidebarOpen = -> 
-    $sidebar.removeClass('sidebar-collapsed').addClass('sidebar-expanded')
-    $sidebar.attr('data-behavior', 'main-sidebar sidebar-expanded')
-    $('[data-behavior~=back-fade]').removeClass('not-faded').addClass('faded')
-
-  sidebarClose = ->
-    $sidebar.removeClass('sidebar-expanded').addClass('sidebar-collapsed')
-    $sidebar.attr('data-behavior', 'main-sidebar sidebar-collapsed')
-    $('[data-behavior~=back-fade]').removeClass('faded').addClass('not-faded')
+  $viewContent = $('[data-behavior~=view-content]')
+
+  sidebarOpen = (animationClass) ->
+    $navbar.css('left', '207px')
+    $sidebar.removeClass('sidebar-collapsed no-animation animate').addClass('sidebar-expanded' + animationClass).attr('data-state', 'sidebar-expanded')
+    $viewContent.css({'left': '250px', 'width': 'calc(100vw - 250px)'})
+    # This is required to keep the sidebar state if the user refreshes the page
+    localStorage.setItem('sidebar-view-state', $sidebar.attr('data-state'));
+
+  sidebarClose = (animationClass) ->
+    $navbar.css('left', '0px')
+    $sidebar.removeClass('sidebar-expanded no-animation animate').addClass('sidebar-collapsed' + animationClass).attr('data-state', 'sidebar-collapsed')
+    $viewContent.css({'left': '43px', 'width': 'calc(100vw - 43px)'})
+    # This is required to keep the sidebar state if the user refreshes the page
+    localStorage.setItem('sidebar-view-state', $sidebar.attr('data-state')); 
+
+  if localStorage.getItem('sidebar-view-state') && localStorage.getItem('sidebar-view-state') == 'sidebar-collapsed'
```

Comment:
```suggestion
  if localStorage.getItem('sidebar-view-state') == 'sidebar-collapsed'
```

You don't need to see if the value exists first. It will return `null` if it does not, and `null` == 'sidebar-collapsed' will be `false`